stories/navbar/navbar.stories.mdx

```diff
@@ -264,6 +267,128 @@ You can also choose to render a custom component of your choice as an action but
+                    items={{
+                        desktop: [
+                            {
+                                id: "home",
+                                children: "Home",
+                                onClick: () => {
+                                    setSelected("home");
+                                },
+                            },
+                            {
+                                id: "guides",
+                                children: "Guides",
+                                onClick: () => {
+                                    setSelected("guides");
+                                },
+                                subMenu: [
+                                    {
+                                        id: "guides2",
+                                        children: "Sub Guides 2",
+                                        onClick: () => {
+                                            setSelected("guides2");
+                                        },
+                                    },
+                                    {
+                                        id: "guides3",
+                                        children: "Sub Guides 3",
+                                        onClick: () => {
+                                            setSelected("guides3");
+                                        },
+                                    },
+                                ],
+                            },
+                        ],
+                        mobile: [
+                            {
+                                itemType: "component",
+                                children: (
+                                    <>
+                                        <CustomComponentWrapper>
+                                            <Text.BodySmall>
+                                                John Smith
+                                            </Text.BodySmall>
+                                            <SubLabel>
+                                                john_smith@tech.gov.sg
+                                            </SubLabel>
+                                            <Button.Small
+                                                style={{
+                                                    marginTop: "1rem",
+                                                    marginBottom: "1rem",
+                                                }}
+                                                onClick={() => {
+                                                    navbarRef.current.dismissDrawer();
+                                                }}
+                                            >
+                                                Click to close the drawer
+                                            </Button.Small>
+                                        </CustomComponentWrapper>
+                                        <hr />
```

Comment:
can trouble you to extract this into a component in `doc-elements.tsx` to keep the story markup simpler